src/PlusDataCollection/vtkPlusUsImagingParameters.cxx

```diff
@@ -466,8 +466,8 @@ PlusStatus vtkPlusUsImagingParameters::ReadConfiguration(vtkXMLDataElement* devi
   for (int i = 0; i < parameterList->GetNumberOfNestedElements(); ++i)
   {
     vtkXMLDataElement* element = parameterList->GetNestedElement(i);
-    std::string name = element->GetAttribute("name") ? element->GetAttribute("name") : "";
-    std::string value = element->GetAttribute("value") ? element->GetAttribute("value") : "";
+    std::string name = element->GetAttribute("Name") ? element->GetAttribute("Name") : "";

```

Comment:
Why this? The write function writes them out in lower case.

Reply:
Thanks for pointing this out. I've added a commit to make each of these (& the UsParameter tag name) a named constant, instead of string literals.